driver-async/src/main/com/mongodb/async/client/AbstractSubscription.java

```diff
@@ -134,9 +140,12 @@ void onError(final Throwable t) {
         }
     }
 
-    void onNext(final TResult next) {
+    private void onNext(final TResult next) {
```

Comment:
`next` should be annotated at `@Nullable` since it now can accept a null value

Reply:
Done